languages/spanish.js

```diff
@@ -242,6 +242,7 @@
 			'e21': 'El formato',
 			'e22': 'no es válido para retos',
 			'e31': 'No tengo equipos para retar en el formato',
+			'spf': 'Por favor especificar un formato', // I'm assuming this is wrong btw :P
```

Comment:
``Por favor especifique un formato`` <-- That would be better, I guess. (I'm spanish)

But, I think that ``Por favor especificar un formato`` could be good too.